x-pack/plugins/apm/e2e/run-e2e.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
`sh` shell might be using `dash` in the CI Workers, let's ensure we do use the `bash` one with the shebang approach. Otherwise, the log traces might indicate something bad is going on:

```
[2020-03-30T14:49:49.373Z] + x-pack/legacy/plugins/apm/e2e/run-e2e.sh
[2020-03-30T14:49:49.373Z] 
[2020-03-30T14:49:49.373Z] To start Kibana please run the following command:
[2020-03-30T14:49:49.373Z] node ./scripts/kibana --no-base-path --dev --no-dev-config --config x-pack/legacy/plugins/apm/e2e/ci/kibana.e2e.yml
[2020-03-30T14:49:49.373Z] 
[2020-03-30T14:49:49.373Z] Temporary folder
[2020-03-30T14:49:49.373Z] Temporary files will be stored in: ./tmp
[2020-03-30T14:49:49.373Z] 
[2020-03-30T14:49:49.373Z] apm-integration-testing (logs: ./tmp/apm-it.log)
[2020-03-30T14:49:49.373Z] Cloning repository
[2020-03-30T14:49:49.373Z] Starting docker-compose
[2020-03-30T14:49:49.373Z] 
[2020-03-30T14:49:49.373Z] Static mock data (logs: ./tmp/ingest-data.log)
[2020-03-30T14:49:49.373Z] Downloading events.json...
[2020-03-30T14:49:49.373Z] x-pack/legacy/plugins/apm/e2e/run-e2e.sh: 63: x-pack/legacy/plugins/apm/e2e/run-e2e.sh: ./tmp/apm-integration-testing/scripts/compose.py: not found
[2020-03-30T14:49:49.373Z] Cloning into './tmp/apm-integration-testing'...
[2020-03-30T14:49:49.373Z] CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
[2020-03-30T14:49:49.373Z] 
[2020-03-30T14:49:49.373Z] ⚠️  Not all events were ingested correctly. This might affect test tests. 
```